inject/inject.vala

```diff
@@ -53,9 +70,11 @@ namespace Frida.Inject {
 		if (script_path == "") {
 			printerr ("Path to JavaScript file must be specified\n");
 			return 3;
+		} else if (script_path == "-") {
```

Comment:
Should also set `script_path` to `null` inside this block – let's do that as the first statement.